help/md/fnWAG.md

```diff
@@ -1,9 +1,28 @@
 ## name
 fnWAG
 ## title
+WAG (Whelan and Goldman) Empirical Amino Acid Substitution Model
 ## description
+The WAG rate matrix derived using both likelihood-based and counting methods for estimating amino acid substitution patterns. 
+
 ## details
+The WAG model, a empirical model of amino acid replacement derived using an approximate maximum-likelihood method from 3,905 sequences across 182 protein families. It outperforms widely used models like Dayhoff and JTT in terms of accuracy and likelihood for phylogenetic analysis, aiming to provide better evolutionary tree estimates and applications in sequence alignment, database searches, and protein structure prediction.
```

Comment:
instead of "widely used models like", maybe "previous models like".  This is because Dayhoff and JTT were widely used in 2001, but are not widely used any more.